apps/web-giddh/src/app/daybook/daybook.component.html

```diff
@@ -713,14 +713,15 @@
                 <!-- pagination -->
                 <div
                     class="mt-1"
-                    [hidden]="!daybookData?.entries?.length || daybookData?.totalItems <= daybookData?.count"
+                    [hidden]="!daybookData?.entries?.length || daybookData?.totalItems > pageSizeOptions[0]"
```

Comment:
```suggestion
                    *ngIf="daybookData?.entries?.length || daybookData?.totalItems > pageSizeOptions[0]"
```

also verify the condition

Reply:
Done